selfdrive/controls/lib/pathplanner.py

```diff
@@ -24,13 +23,12 @@ def calc_states_after_delay(states, v_ego, steer_angle, curvature_factor, steer_
 
 class PathPlanner(object):
   def __init__(self, CP):
-    self.LP = LanePlanner()
+    self.LP = LanePlanner(True)
```

Comment:
At the point of use, here, it's not clear what True/False represent, consider specifying the argument name.
```python
self.LP = LanePlanner(withALCA=True)
```